app/models/project.rb

```diff
@@ -185,6 +185,10 @@ def download_url(version = nil)
     platform_class.download_url(name, version)
   end
 
+  def latest_download_url
+    download_url(latest_release_number)
```

Comment:
To me the default value of `nil` for the version param in `download_url` doesn't make logical sense and it doesn't look like the platform_class implementations handle it properly. But since it's there and we handle a nil value here anyway (for package managers that don't support download_url) I went ahead and returned nil for the url in that case.

I'm not sure latest_release_number can actually be nil (can we have packages with no releases?) but it would now eval to nil if that happens, which I think is most sensible. This field is allowed to be nil already due to package managers that don't support it.